src/pages/home/report/EmojiPickerMenu/index.js

```diff
@@ -72,6 +74,9 @@ class EmojiPickerMenu extends Component {
             headerIndices: this.unfilteredHeaderIndices,
             highlightedIndex: -1,
             arePointerEventsDisabled: false,
+            emojiSize: {
+                fontSize: dynamicEmojiSize(this.props.windowWidth),
+            },
```

Comment:
Why this?

To reduce the memory overhead of creating objects for each emoji.

Reply:
Nice